bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx68.html

```diff
@@ -0,0 +1,85 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_67" %}
+
+{% extends "firefox/whatsnew/whatsnew-fx67.html" %}
+
+{% block fxa_form %}
+  {{ fxa_email_form(
+      entrypoint='mozilla.org-whatsnew67',
```

Comment:
Should this be `mozilla.org-whatsnew68` for entrypoint?

Reply:
Good catch 👍